cli/docker/docker.go

```diff
@@ -26,6 +26,34 @@ func ConnectContainerToDevNetwork(containerID string, aliases ...string) error {
 		})
 }
 
+// ExecCommandIntoContainer executes a command, as a user, into a container
+func ExecCommandIntoContainer(containerName string, user string, cmd []string) error {
+	dockerClient := getDockerClient()
+
+	response, err := dockerClient.ContainerExecCreate(
+		context.Background(), containerName, types.ExecConfig{
```

Comment:
Add the context as parameter of this function, or use `context.TODO()`.

Reply:
You mean `ExecCommandIntoContainer` receiving a context, right?

Reply:
Applied